.gitignore

```diff
@@ -1,5 +1,7 @@
 npm-debug.log
 .project
+package-lock.json
```

Comment:
Why are you ignoring the package-lock?

Reply:
Because I hate it. I no longer buy the reasons for committing it, since npm is slow as molasses in any case, and it adds huge volume to diffs which are seen at the console and in review UIs where the changes consist just of a single line update, for example as seen in https://github.com/GPII/gpii-express/pull/39/files .
Those people in the package management development teams who advocate checking in machine generated files should be first against the wall when the revolution comes.

Reply:
We should discuss it in a wider forum, as we've already transitioned to doing it in many of our core libraries.  Rather than waiting for the revolution and my own spot on the wall, I'll bring it up in tonight's architecture meeting so you can make the case.